nestkernel/nest_names.cpp

```diff
@@ -25,6 +25,9 @@ namespace nest
 {
 namespace names
 {
+const Name Ca( "Ca" );
```

Comment:
The names list should be kept in alphabetical order (applies also to the .h file).